### PR TITLE
[Chore] Get expression unit from the DB for Cell plots tab / Gene Expression part

### PR DESCRIPTION
**Pull request by ke4**
Currently the `expression unit` of an experiment is hardcoded everywhere to `CPM`.
This change is going to fix it in the cell plot diagram (in the tooltip).
We are already storing the value of the expression unit in the DB, in the `experiment` table.
We are going to get it from there and pass it to the UI.

I tested it locally and it work fine. 

This is fixing the 6th case in this ticket: [Make experiment unit adaptable not hard-coded in marker gene heatmap](https://github.com/ebi-gene-expression-group/atlas-web-single-cell/issues/447)	https://github.com/ebi-gene-expression-group/atlas-web-single-cell/issues/447
According to Silvie this was the highest priority.
We need to create other PRs for the other cases.

### CODE REVIEW

**Review by lingyun1010 — DISMISSED** [review]
LGTM!

**Comment by ke4** [comment]
@lingyun1010 is already approved this PR, but after her approval I accidentally pushed another commit to this PR as it was a very similar case, but I realised that it is going to need another PR, so I removed that commit from this PR.
I am going to force `Squash and merge` this PR now.
Sorry about this.

**Pull request merged**
